beatrix/pom.xml

```diff
@@ -295,6 +295,10 @@
             <groupId>org.kill-bill.commons</groupId>
             <artifactId>killbill-jdbi</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.kill-bill.commons</groupId>
+            <artifactId>killbill-locker</artifactId>
```

Comment:
Technically `<scope>test</scope>` although this might confuse Maven (since it's also required via runtime transitive dependencies).

Reply:
Yeah, I'd rather not touch it :-)